src/main/java/org/isf/lab/model/Laboratory.java

```diff
@@ -106,10 +106,21 @@ public class Laboratory extends Auditable<String> {
 
 	@Column(name="LAB_SEX")
 	private String sex;
+	
+	@Column(name="LAB_STATUS")
+	private String status;
```

Comment:
@dbmalkovsky for GUI should we assign a default value when creating a laboratory (directly in "done" I suppose)

Reply:
why not just create the default for status when creating the object in the constructors in `Laboratory.java`?

Reply:
I mean, for GUI the field is not (yet) managed, so the default (that should be "draft" in the constructor) for GUI has to be set to "done". So I think a PR on GUI is needed to set this field, wdyt?

Reply:
Example here https://github.com/informatici/openhospital-gui/pull/1736